### PR TITLE
Feat #123 Version display and add Dockerfile for test

### PR DESCRIPTION
**Pull request by KEINOS**
I **added a simple version display feature** of issue #123. (c0d754d)
and **Dockerfile for non-golang-installed users**. (9432b56)

First of all, I couldn't find a way to pull the `git tag`'s version info while `go get`. 

So I did in an ordinary way. Such as giving the version number via `-X` in `--ldflags` building option.
Whith this, it will set the `versionKagome` variable inside the `kagome/cmd/main.go`.

```shellsession
$ cd /path/to/git/kagome/cmd
$ # Get version from git tag
$ version_app=$(git describe --tag)
$ # Build kagome specifying the version
$ go build --ldflags "-X 'main.versionKagome=${version_app}'" -o /go/bin/kagome ./kagome
...
$ /go/bin/kagome version
kagome v1.9.0-2-g1b1efdb
```

Also, I used Docker to check its work on Linux image. 9432b56
And the [build-test (`docker-compose.test.yml`) on Docker Hub](https://hub.docker.com/r/keinos/kagome) did pass. 

```shellsession
$ cd /path/to/git/kagome
$ docker build --tag kagome:latest .
...
$ docker run --rm kagome version
kagome v1.9.0-2-g1b1efdb
$
$ docker run --rm -it kagome tokenize
すもももももももものうち
すもも	名詞,一般,*,*,*,*,すもも,スモモ,スモモ
も	助詞,係助詞,*,*,*,*,も,モ,モ
もも	名詞,一般,*,*,*,*,もも,モモ,モモ
も	助詞,係助詞,*,*,*,*,も,モ,モ
もも	名詞,一般,*,*,*,*,もも,モモ,モモ
の	助詞,連体化,*,*,*,*,の,ノ,ノ
うち	名詞,非自立,副詞可能,*,*,*,うち,ウチ,ウチ
EOS

```

With this, we will be able to provide `kagome` for no-golang-installed-user via Docker.

And if you link the GitHub repo to the Docker Hub's repo, people can use `kagome` by:

```shellsession
$ docker pull ikawaha/kagome:latest
...
$ docker run --rm -it ikawaha/kagome tokenize
すもももももももものうち
すもも	名詞,一般,*,*,*,*,すもも,スモモ,スモモ
も	助詞,係助詞,*,*,*,*,も,モ,モ
もも	名詞,一般,*,*,*,*,もも,モモ,モモ
も	助詞,係助詞,*,*,*,*,も,モ,モ
もも	名詞,一般,*,*,*,*,もも,モモ,モモ
の	助詞,連体化,*,*,*,*,の,ノ,ノ
うち	名詞,非自立,副詞可能,*,*,*,うち,ウチ,ウチ
EOS

```

Though, I should've issued about Docker before the PR. Please drop the commit 9432b56 if Docker is not necessary. Or drop the hole PR if we need to discuss it in the issue.

### CODE REVIEW

File: cmd/kagome/main.go [hunk 1]
```diff
@@ -28,6 +28,8 @@ import (
 var (
 	errorWriter = os.Stderr
 
+	version_kagome string
```
Comment (ikawaha) [inline cmd/kagome/main.go]: __[Golint]__ don't use underscores in Go names; var version_kagome should be versionKagome ([link](https://sider.review/gh/repos/21228174/pulls/149#issue-4158057))

<sub>posted by Sider</sub>

Comment (KEINOS) [reply cmd/kagome/main.go]: (￣Λ￣)ゞ AI AI SIR!

**Comment by ikawaha** [comment]
Thank you for the PR! It's great!
Since there is diffs waiting for merging in the develop branch, 
please direct the merge destination to the `develop`.

**Comment by KEINOS** [comment]
@ikawaha 
Glad to hear that! Then it's been worth doing it!
I switched the base branch pointer from `master` to `develop`!!

**Comment by KEINOS** [comment]
Seems that the CI didn't run when changed the base branch, I'll commit a typo fix which I just found.

**Comment by ikawaha** [comment]
Thanks!

**Pull request merged**